.rubocop.yml

```diff
@@ -48,3 +48,119 @@ Rails/Output:
 Rails/Exit:
   Exclude:
     - lib/decidim/git_backport_manager.rb
+
+# fix these rules later
```

Comment:
Just a question, are you planning to fix these in the near future? So we separate updating the dependencies to this PR and then another PR to fix the Rubocop rules?